site/en/stories/stories.json

```diff
@@ -1,6 +1,5 @@
 {
   "layout": "article",
-  "tags": ["case study"],
```

Comment:
This line needs to be added back or none of the case studies show up on the page